resources/META-INF/plugin.xml

```diff
@@ -0,0 +1,30 @@
+<idea-plugin url="https://github.com/strawberry-graphql/strawberry-pycharm-plugin" require-restart="true">
+    <id>rocks.strawberry</id>
+    <name>Strawberry-Graphql</name>
+    <version>0.0.1</version>
+    <vendor email="koaxudai@gmail.com">TODO Koudai Aono @koxudaxi</vendor>
```

Comment:
will this email be used for support? we can create one @strawberry.rocks if needed :)

Reply:
great!!
This plugin.xml will be a detailed page in the marketplace.
https://plugins.jetbrains.com/plugin/12861-pydantic

Reply:
got it, we can update the email later :)